pepper-apis/src/main/java/org/broadinstitute/ddp/log/RedisConnectionValidatorAppender.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.ddp.log;
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.AppenderBase;
+import org.broadinstitute.ddp.util.RedisConnectionValidator;
+
+public class RedisConnectionValidatorAppender extends AppenderBase<ILoggingEvent> {
+    @Override
+    protected void append(ILoggingEvent eventObject) {
+        RedisConnectionValidator.doTest();
+    }
+}
```

Comment:
Not a real appender. But if Redisson library logs an ERROR (particularly the libraries own PINGs which we do not call explicitly), we will do our own ping of Redis.